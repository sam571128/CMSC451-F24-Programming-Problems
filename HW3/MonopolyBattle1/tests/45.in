22 41 443
12 13
12 6
9 3
2 21
9 17
13 5
22 4
6 17
8 5
4 7
10 12
7 11
2 16
2 3
21 13
7 7
14 13
21 19
14 20
15 18
20 17
15 5
13 3
12 2
9 1
5 3
8 1
7 17
2 1
18 21
19 8
21 18
11 14
6 16
10 13
11 5
18 15
6 15
6 6
11 20
12 15
