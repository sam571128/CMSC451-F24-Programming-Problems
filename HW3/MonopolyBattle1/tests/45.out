7771 8617 8617 8617 8617 8654 8712 8848 8617 8617 8694 8682 8617 8617 7890 8617 8617 8617 8617 7890 8617 8712 